Common/Sources/View/Error/ErrorView.swift

```diff
@@ -76,10 +76,20 @@ public final class ErrorView {
     }
 
     private func unexpectedError() {
+      if let pageName = self.pageName, let pageGroup = self.pageGroup {
+       let errorInfo = ErrorInfoBuilder(type: self.errorTypeException,
+                                        message: self.error.errorDescription ?? "")
```

Comment:
message should be `error.getHyperwalletErrors()?.errorList?.first?.message`

Reply:
Message was changed